docs/content/getting-started/setup-openfga/overview.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Setup OpenFGA
+description: Setting up an OpenFGA server
+sidebar_position: 1
+slug: /getting-started/setup-openfga/overview
+---
+
+import {
+  DocumentationNotice,
+  RelatedSection,
+  CardGrid,
+} from "@components/Docs";
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Setup OpenFGA
+Follow the guides below to setup an OpenFGA server.
+
+<DocumentationNotice />
+
+<CardGrid
+  middle={[
+    {
+      title: 'Docker Setup Guide',
+      description: 'How to setup an OpenFGA server with Docker.',
+      to: 'docker',
+    },
+    {
+      title: 'Kubernetes Setup Guide',
+      description: 'How to setup an OpenFGA server with Kubernetes.',
+      to: 'kubernetes',
```

Comment:
```suggestion
      to: 'getting-started/setup-openfga/kubernetes',
```

Reply:
Same comment as in https://github.com/openfga/openfga.dev/pull/402#discussion_r1152421424